src/main/java/entity/User.java

```diff
@@ -0,0 +1,159 @@
+package entity;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class User {
+
+    private int id;
+    private int following;
+    private int followers;
+    private List<Post> postList;
+    private String name;
+    private String password;
+    private Date joinDate;
+    private int totalLikes;
+
+    private User(Builder builder) {
+        this.id = builder.id;
+        this.following = builder.following;
+        this.followers = builder.followers;
+        this.postList = builder.postList;
+        this.name = builder.name;
+        this.password = builder.password;
+        this.joinDate = builder.joinDate;
+        this.totalLikes = builder.totalLikes;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public int getFollowing() {
+        return following;
+    }
+
+    public void setFollowing(int following) {
+        this.following = following;
+    }
+
+    public int getFollowers() {
+        return followers;
+    }
+
+    public void setFollowers(int followers) {
+        this.followers = followers;
+    }
```

Comment:
Instead of set, it should be `addFollower` and `addFollowing` once they are lists.